skbio/metadata/tests/test_intersection.py

```diff
@@ -0,0 +1,226 @@
+# ----------------------------------------------------------------------------
```

Comment:
Support is now available to ignore this file's copyright header validation (#1422). Please remove the copyright here and add this line as the first line in the file:

```
# checklist.py:CopyrightHeadersValidator IGNORE
```